src/framework/core/js/Fluid.js

```diff
@@ -826,6 +826,28 @@ var fluid = fluid || fluid_3_0_0;
         return isFinite(string) && ((string % 1) === 0) ? Number(string) : NaN;
     };
 
+    /**
+     * Derived from Sindre Sorhus's round-to node module ( https://github.com/sindresorhus/round-to ).
+     * License: MIT
+     *
+     * Rounds the supplied number to at most the number of decimal places indicated by the scale, omiting any trailing 0s.
+     * Numbers are rounded away from 0 (i.e 0.5 -> 1, -0.5 -> -1).
+     * If the scale is invalid (i.e falsey, not a number, negative value), it is treated as 0.
+     * If the scale is a floating point number, it is rounded to an integer.
+     *
+     * @param {Number} num - the number to be rounded
+     * @param {Number} scale - the maximum number of decimal places to round to.
+     * @return {Number} The num value rounded to the specified number of decimal places.
+     */
+    fluid.roundToDecimal = function (num, scale) {
+        // treat invalid scales as 0
+        scale = scale && scale >= 0 ? Math.round(scale) : 0;
+
+        // The following is derived from https://github.com/sindresorhus/round-to/blob/v2.0.0/index.js#L17
+        var sign = num >= 0 ? 1 : -1; // manually calculating the sign because Math.sign is not supported in IE
```

Comment:
Isn't it incredible that this is still not supported even in IE11! What an appalling browser, it will be great when we can ditch support for this.